test-files/initVaultDev.sh

```diff
@@ -1,7 +1,23 @@
-#!/bin/bash
-export VAULT_ADDR=http://localhost:8200
-export VAULT_TOKEN=my-dev-root-vault-token
-export VAULT_VERSION=${1:-1.4.1}
+#!/usr/bin/env bash
+# shellcheck disable=SC2155
+
+declare -r VAULT_ADDR=http://localhost:8200
+declare -r VAULT_TOKEN=my-dev-root-vault-token
+declare -r VAULT_VERSION=${1:-1.4.1}
+
+# Where are we?
+declare -r SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" >/dev/null 2>&1 && pwd  )"
+declare -r VAULT_BIN_DIR="${SCRIPT_DIR}/vault"
+declare -r VAULT_BIN="${VAULT_BIN_DIR}/vault${VAULT_VERSION}"
+declare -r VAULT_LOG_DIR="${SCRIPT_DIR}/logs"
+declare -r VAULT_LOG="${VAULT_LOG_DIR}/vault${VAULT_VERSION}.log"
+declare -r VAULT_PID="${SCRIPT_DIR}/vault.pid"
+
+# Check for leftover processes from previous run
+if [[ -f "${VAULT_PID}" ]]; then
+  echo "There is a pid file present from a previous run. Aborting!"
+  exit 1
+fi
```

Comment:
This block is not consistently formatted compared to the block below.

Reply:
What do you mean by formatted?

Reply:
The block from lines 40 to 43 is indented differently than line 18 and 19